src/pages/Dashboard.vue

```diff
@@ -194,12 +194,13 @@ onBeforeUnmount(() => {
 
     <v-row class="ma-0 pa-0">
       <v-expand-transition>
-        <v-card v-show="isSelectionMultiple" color="transparent">
+        <v-card v-show="isSelectionMultiple" color="transparent" variant="flat">
```

Comment:
```suggestion
        <v-card v-show="isSelectionMultiple" color="transparent" flat>
```

You can also use the flat prop which does the exact same thing

Reply:
Thanks, good to know that. I'm completely new to Vue and its ecosystem 🙂